bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -103,6 +101,8 @@ public class CommunityMarketplaceAddonService extends AbstractRemoteAddonService
     private static final Integer RULETEMPLATES_CATEGORY = 74;
     private static final Integer UIWIDGETS_CATEGORY = 75;
     private static final Integer BLOCKLIBRARIES_CATEGORY = 76;
+    // TODO: confirm category
+    private static final Integer TRANSFORMATIONS_CATEGORY = 77;
```

Comment:
```suggestion
    private static final Integer TRANSFORMATIONS_CATEGORY = 80;
```
See https://community.openhab.org/c/marketplace/transformations/